internal/manifests/collector/deployment.go

```diff
@@ -38,6 +39,31 @@ func Deployment(params manifests.Params) (*appsv1.Deployment, error) {
 		return nil, err
 	}
 
+	initContainers := params.OtelCol.Spec.InitContainers
+
+	if params.Config.CertManagerAvailability() == certmanager.Available {
+		initContainers = append(initContainers, corev1.Container{
```

Comment:
We need to add the ca.crt cert to the client trust store, and from what i understand it is best to let `update-ca-certificates` do that as opposed to touch the `/etc/ssl/certs/ca-certificates.crt` file directly.

Reply:
If we do want the CA cert here, do we need to merge it with the existing store, even? Neither side of the collector -> ta connection needs to to accept any other certs.

Reply:
Not that i know of, but then why would it be there to begin with? maybe for outgoing endpoints? exporters?

Reply:
For the collector, so it can connect to the outside world. For target allocator I don't think it's necessary right now, but there's no harm in including it.